mlrun/utils/helpers.py

```diff
@@ -1718,6 +1718,44 @@ def validate_single_def_handler(function_kind: str, code: str):
             )
 
 
+def validate_node_selectors(node_selectors: dict) -> None:
```

Comment:
```suggestion
def validate_node_selectors(node_selectors: typing.Dict[str, str]) -> None:
```

Reply:
linter raised a warning for `typing.Dict[str, str]`, I used `dict[str, str]` instead